working_with_hl7/working_with_hl7_parsing.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import wso2healthcare/healthcare.hl7 as hl7;
```

Comment:
shall we change to ballerinax dependencies and check since the new changes are released?

Reply:
updated.